onnxscript/optimizer/_constant_folding.py

```diff
@@ -40,21 +35,21 @@ def is_control_flow_op(node: ir.Node) -> bool:
 )
 
 
-def is_non_deterministic_op(node: ir.Node) -> bool:
-    return node.op_type in non_deterministic_ops and utils.is_onnx_domain(node.domain)
+logger = logging.getLogger(__name__)
 
 
-def is_onnx_op(node: ir.Node, op_type: str) -> bool:
-    return node.op_type == op_type and utils.is_onnx_domain(node.domain)
+def _is_control_flow_op(node: ir.Node) -> bool:
+    graph_types = {ir.AttributeType.GRAPH, ir.AttributeType.GRAPHS}
+    return any(attr.type in graph_types for attr in node.attributes.values())
 
 
-def is_constant_op(node: ir.Node) -> bool:
-    return node.op_type in {"Constant", "ConstantOfShape"} and utils.is_onnx_domain(
-        node.domain
-    )
+def _is_non_deterministic_op(node: ir.Node) -> bool:
+    return node.op_type in _NON_DETERMINISTIC_OPS and utils.is_onnx_domain(node.domain)
```

Comment:
Minor: Now that we normalize domains, the utils.is_onnx_domain can be simplified or replaced by comparison with "".

Reply:
I am ok with leaving the util to make it more readable. Does that work for you?